rest/_examples/alerts.go

```diff
@@ -0,0 +1,132 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"time"
+
+	api "gopkg.in/ns1/ns1-go.v2/rest"
+	"gopkg.in/ns1/ns1-go.v2/rest/model/alerting"
+	"gopkg.in/ns1/ns1-go.v2/rest/model/dns"
+	"gopkg.in/ns1/ns1-go.v2/rest/model/monitor"
+)
+
+var client *api.Client
+
+// Helper that initializes rest api client from environment variable.
+func init() {
```

Comment:
Is this called in the example somewhere?

Reply:
If I'm 100% honest this is here because I started by copying one of the other examples and wanted it to follow our existing pattern.
It will always be called as go ensures the `init()` function for a file is always run:
https://go.dev/doc/effective_go#init
(The internet seems rather divided on whether it's a bad thing to use which in general but it's probably ok for samples like this.)